fs/glob.ts

```diff
@@ -86,9 +82,29 @@ export function isGlob(str: string): boolean {
 
 export interface ExpandGlobOptions extends GlobOptions {
   root?: string;
+  exclude?: string[];
   includeDirs?: boolean;
 }
 
+const SEP_PATTERN = isWindows ? /[\\/]+/ : /\/+/;
```

Comment:
Maybe we should have this constant in "./path/constants.ts"?

Reply:
👍